test/system/research/join_experiment_flow_test.rb

```diff
@@ -0,0 +1,28 @@
+require_relative "./test_case"
+
+module Research
+  class JoinExperimentFlowTest < Research::TestCase
+    test "user joins an experiment from the experiments page" do
+      user = create(:user, :onboarded)
+      experiment = create(:research_experiment)
+
+      sign_in!(user)
+      visit research_experiments_path
+      click_on "Participate"
+
+      assert UserExperiment.where(user: user, experiment: experiment).exists?
```

Comment:
I think we should assert at the integration level, preferably checking text exists on this page, etc.

Reply:
I sort of agree. But we don't have any text yet that won't change, and I didn't really want to tightly-couple the test to the wrong text. Thoughts?